lib/GameConstants.h

```diff
@@ -440,9 +440,9 @@ namespace ESpellCastProblem
 {
 	enum ESpellCastProblem
 	{
-		OK, NO_HERO_TO_CAST_SPELL, ALREADY_CASTED_THIS_TURN, NO_SPELLBOOK, ANOTHER_ELEMENTAL_SUMMONED,
+		OK, NO_HERO_TO_CAST_SPELL, ALREADY_CAST_THIS_TURN, NO_SPELLBOOK,

```

Comment:
I think we could someday have an option to alter number of casts per turn (or at least for casting creatures). Might be some more neutral name can be used.